packages/libs/chainwebjs/src/tests/mocks/360hashes.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-lines */
```

Comment:
Let's add the `tests/mocks` directory to `packages/libs/chainwebjs/.eslintignore`